src/vaadin-message-input-text-area.js

```diff
@@ -59,9 +68,9 @@ class MessageInputTextAreaElement extends TextAreaElement {
     });
   }
 
-  _setAriaLabel(message) {
+  __ariaLabelChanged(ariaLabel) {
     // Set aria-label to provide an accessible name for the labelless input
-    this.inputElement.setAttribute('aria-label', message);
+    this.inputElement.setAttribute('aria-label', ariaLabel);
```

Comment:
This will set `aria-label`  to `undefined` by default. Maybe we could check if `ariaLabel` is truthy.